src/config.rs

```diff
@@ -239,6 +239,7 @@ pub struct Vagrant {
 #[serde(deny_unknown_fields)]
 pub struct Windows {
     accept_all_updates: Option<bool>,
+    updates_auto_reboot: Option<bool>,
```

Comment:
I think there should be an option to leave out the explicit policy and have it ask:
```rust
enum UpdatesAutoReboot {
    Yes,
    No,
    Ask,
}
```
I'm personally fine with the default still being `"no"`, but there should at least be the option to revert to the previous behavior.